ReactAndroid/build.gradle

```diff
@@ -46,17 +46,13 @@ def JSCPath = '../jsc'
 def V8Path = 'packages/Office.Google_V8.Android.7.0.276.32-v1'
 
 def hasBoostNuget = file(BoostNugetPath).exists()
+def JSEngineUsed = "JSC"
```

Comment:
>JSC [](start = 20, length = 3)

1. Default engine should be V8
2. Engine name should be in single quotes.
3. All Engine name should be defined above this statement.